utils/jmx.py

```diff
@@ -8,15 +8,48 @@
 import yaml
 
 # datadog
-from config import _windows_commondata_path
-from util import yDumper
+from config import _windows_commondata_path, get_confd_path
+from util import get_os, yDumper
```

Comment:
Can you remove this import ? (`get_os`) If `get_confd_path` doesn't have a `get_os()`, it will call it itself, so it's no necessary.